pkg/qemu/qemu.go

```diff
@@ -372,6 +373,17 @@ func Cmdline(cfg Config) (string, []string, error) {
 
 	// We also want to enable vsock and virtfs here, but QEMU does not support vsock and virtfs for macOS hosts
 
+	if y.MountType == "9p" {
+		for i, f := range y.Mounts {
+			expanded, err := localpathutil.Expand(f.Location)
+			if err != nil {
+				return "", nil, err
+			}
+			tag := fmt.Sprintf("mount%d", i)
+			args = append(args, "-virtfs", fmt.Sprintf("local,path=%s,mount_tag=%s,security_model=mapped-xattr", expanded, tag))
```

Comment:
Probably you have to quote path

Reply:
Not sure what is the best function, to escape paths before passing as args (i.e. space or quote is not an issue here, but any commands in the location might be...)

Reply:
Leaving this for now, since it is an argument (and not shell) it should at least survive normal characters ?